### PR TITLE
Package promise_jsoo.0.1.0

### PR DESCRIPTION
**Pull request by mnxn**
### `promise_jsoo.0.1.0`
Js_of_ocaml bindings to JS Promises with supplemental functions

---
* Homepage: https://github.com/mnxn/promise_jsoo
* Source repo: git+https://github.com/mnxn/promise_jsoo.git
* Bug tracker: https://github.com/mnxn/promise_jsoo/issues

---
:camel: Pull-request generated by opam-publish v2.0.2

### CODE REVIEW

**Comment by mnxn** [comment]
FreeBSD build fails with
```
ld-elf.so.1: /usr/local/bin/node: Undefined symbol "nghttp2_option_set_max_settings"
```
which is a FreeBSD problem:
https://stackoverflow.com/q/62873601
https://bugs.freebsd.org/bugzilla/show_bug.cgi?id=241643

File: packages/promise_jsoo/promise_jsoo.0.1.0/opam [hunk 1]
```diff
@@ -0,0 +1,39 @@
+opam-version: "2.0"
+synopsis: "Js_of_ocaml bindings to JS Promises with supplemental functions"
+maintainer: ["Max Lantas <mnxn.dev@outlook.com>"]
+authors: ["Max Lantas <mnxn.dev@outlook.com>"]
+license: "MIT"
+homepage: "https://github.com/mnxn/promise_jsoo"
+bug-reports: "https://github.com/mnxn/promise_jsoo/issues"
+depends: [
+  "dune" {>= "2.7"}
```
Comment (mseri) [inline packages/promise_jsoo/promise_jsoo.0.1.0/opam:9]:
```suggestion
  "ocaml" {>= "4.08"}
```
From the tests it seems to support only ocaml versions more recent than 4.08

Comment (mseri) [reply packages/promise_jsoo/promise_jsoo.0.1.0/opam:9]: Can you upstream this change?

Comment (mnxn) [reply packages/promise_jsoo/promise_jsoo.0.1.0/opam]: Added the ocaml version constraint.

Why was the dune dependency dropped?

Comment (mseri) [reply packages/promise_jsoo/promise_jsoo.0.1.0/opam]: My  mistake sorry, I meant to append!

File: packages/promise_jsoo/promise_jsoo.0.1.0/opam [hunk 2]
```diff
@@ -0,0 +1,39 @@
+opam-version: "2.0"
+synopsis: "Js_of_ocaml bindings to JS Promises with supplemental functions"
+maintainer: ["Max Lantas <mnxn.dev@outlook.com>"]
+authors: ["Max Lantas <mnxn.dev@outlook.com>"]
+license: "MIT"
+homepage: "https://github.com/mnxn/promise_jsoo"
+bug-reports: "https://github.com/mnxn/promise_jsoo/issues"
+depends: [
+  "ocaml" {>= "4.08"}
```
Comment (mseri) [inline packages/promise_jsoo/promise_jsoo.0.1.0/opam:9]:
```suggestion
  "ocaml" {>= "4.08"}
  "dune" {>= "2.7"}
```

Comment (mseri) [reply packages/promise_jsoo/promise_jsoo.0.1.0/opam:9]: As @mnxn correctly pointed out, I had dropped dune in the previous commit

**Comment by mseri** [comment]
I'll wait for the CI to run, but I think this should be fine to merge now

**Comment by mseri** [comment]
Feel free to announce the release on the discuss.ocaml.org forum. We have a community caterogy and an announcement tag for this purpose

**Pull request merged**